.bazelignore

```diff
@@ -21,6 +21,7 @@ third_party/upb
 third_party/xds
 
 bazel/test/python_test_repo
```

Comment:
the python_test_repo here is basically a bazel distribtest for grpc python, right?
We should make the path to python and c++ bazel distribtests consistent. `bazel/test/python_test_repo` and `test/distrib/bazel/cpp` seems quite random.
Can you at least add a TODO to make this consistent?